src/store/models/account.ts

```diff
@@ -115,6 +115,7 @@ export default createModel<RootModel>()({
         .catch(e => setError(e.message));
     },
     async logout() {
+      dispatch.wallet.SET_ASSETS(null);
```

Comment:
I just tried this, it does not seem to be solving the issue, since I still see previous transactions hanging around for a few seconds before they are swapped.